src/components/Request/index.js

```diff
@@ -0,0 +1,66 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import PropTypes from 'prop-types'
+import { Color, Space, Radius } from 'lib/theme'
+import { numericString } from 'lib/utils/strings'
+import { formatDate } from 'lib/utils/datetime'
+import { numberWithCommas } from 'lib/utils/number'
+import { SecondaryButton } from 'components/Button'
+import Text from 'components/Text'
+import { TEXT_TYPE } from 'components/Text/constants'
+import { styles } from './Request.styles'
+
+export const Request = ({ date, donation, id, name, onDelete, request }) => {
+  const numberDonation = numericString(donation)
+  const numberRequest = numericString(request)
+  const progress = (numberDonation / numberRequest) * 100
+  return (
+    <div css={styles.root}>
+      <div css={styles.section}>
+        <Text as="p" type={TEXT_TYPE.NOTE} css={styles.content}>
+          Request ID #{id}
+        </Text>
+        <SecondaryButton onDelete={onDelete}>
+          <Text type={TEXT_TYPE.NOTE}>Delete</Text>
+        </SecondaryButton>
+      </div>
+      <div css={styles.requestName}>
+        <Text>{name}</Text>{' '}
+        <Text css={styles.amount}>{numberWithCommas(numberRequest)}</Text>
+      </div>
+      <div css={[styles.section, styles.progressInfo]}>
+        <Text type={TEXT_TYPE.NOTE} css={styles.content}>
+          {progress ? `${numberDonation} donations` : 'No donations yet'}
+        </Text>
+        <Text type={TEXT_TYPE.NOTE} css={styles.content}>
+          {progress}% Completed
+        </Text>
+      </div>
+      <div css={styles.progressBar}>
+        <div
+          css={styles.progress}
+          css={{
+            background: Color.CORAL,
+            borderRadius: Radius.ROUNDED,
+            height: Space.S10,
+            width: `${progress}%`,
```

Comment:
can you add the static attrs to styles.progress and then compose?

`css={[styles.progress, { width: `${progress}`% }]}

Reply:
@izgeorge i tried this initially and it didn't work :(

Reply:
🤔  weird, did you try wrapping it with the css func?

Reply:
that's a good idea! let me try that

Reply:
sike! that worked! thank you <3